packages/utils/.eslintrc

```diff
@@ -0,0 +1,5 @@
+{
+  "rules": {
+    "import/no-default-export": "error"
+  }
+}
```

Comment:
You can apply this rule in the `.eslintrc` file at the root of the repository, this way, we'll avoid making the same mistake in other packages. If there are many conflicts to fix in order to apply this rule globally, we could do it in a subsequent PR, that's OK.

Reply:
I created an issue to enforce `no-default-export` for all Request Network packages. https://github.com/RequestNetwork/requestNetwork/issues/1034